electron/createWindow.js

```diff
@@ -164,5 +164,15 @@ export default appState => {
     shell.openExternal(url);
   });
 
+  window.webContents.on('update-target-url', (event, url) => {
+    // Change internal links to the lbry protocol. External (https) links should remain unchanged.
+    let dispUrl = url.replace(`http://localhost:${WEBPACK_ELECTRON_PORT}/`, lbryProto);
+    if (dispUrl.startsWith(lbryProto + '$/')) {
+      // Non-claims don't need the lbry protocol.
+      dispUrl = dispUrl.replace(lbryProto, '/');
+    }
```

Comment:
I think this also needs a line that just returns "Home"

Right now I see "lbry://"